.github/workflows/update.yml

```
@@ -27,13 +56,23 @@ jobs:
         release_name: Release ${{ steps.date.outputs.date }}
         draft: false
         prerelease: false
-    - name: Upload Release Asset
-      id: upload-release-asset
+    - name: Upload Linux Release Asset
+      id: upload-linux-asset
+      uses: actions/upload-release-asset@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: linux-index/files
+        asset_name: linux-index
```

If break compatibility maybe we should just include the architecture as well:

```suggestion
        asset_name: index-x86_64-linux
```